build.psm1

```diff
@@ -539,6 +539,12 @@ Fix steps:
         }
     }
 
+    if ($Environment.IsUbuntu)
+    {
```

Comment:
Is this symbolic link required on all Ubuntu versions? We support 14.04 16.04 and 17.04
Is it also required on `Debian 8/9`? We support `Debian 8/9`

Reply:
I don't see an answer to this question

Reply:
@daxian-dbw I was not aware libgssapi_krb5 has an issue on Debian - is that true or are you refering to openssl?  Just want to clarify.

Reply:
I checked with the OMI team.  Debian 9 does not need it. They don't have Debian 8 in their testing so it's unknown.

Reply:
Debian 8 also needs the symbolic link.  I'll update the PR.

Reply:
@daxian-dbw and @TravisEz13 - I've updated the PR to create the symbolic link on Debian.